test/run

```diff
@@ -16,7 +16,7 @@ source $(dirname $0)/common
 
 cd ${ROOT}
 
-extra_args="--ursula-user=${LOGIN_USER} --ursula-debug --provisioner=heat --heat-stack-name=${testenv_heat_stack_name}"
+extra_args="--ursula-user=${LOGIN_USER} --ursula-debug --ursula-sudo"
```

Comment:
the reason to remove it?

- It is no necessary. The predeploy phase will create the stack. need not to call heat in the run phase. test and rally phase .
- Too many call to tardis os api, it will introduce some failure of CI once tardis did not respond correctly.
- It will reduce unnecessary time consuming for CI